frontend/server/src/DAO/Submissions.php

```diff
@@ -313,4 +313,26 @@ public static function getLatestSubmissions(
             'submissions' => $submissions,
         ];
     }
+
+    /** @return \OmegaUp\Timestamp|null */
+    public static function getLastSubmissionByProblem(
+        int $problemId,
+        ?int $identityId
+    ) {
+        $params = [$problemId, $identityId];
+        $sql = '
+            SELECT
+                s.time
+            FROM
+                Submissions s
+            WHERE
+                s.problem_id = ? AND s.identity_id = ?
+            ORDER BY
+                s.time DESC
```

Comment:
```suggestion
                s.submission_id DESC
```

aquí esto es _ligeramente_ más rápido.

Reply:
Va